### DISCUSSION TITLE
使用 TxCaptchaHelper 的时候不知道怎么返回ticket

### DISCUSSION THREAD

**Discussion by Robin-sys** [mamoe/mirai-console#355] [问答]
因为selenium的插件没有用了，所以配置的时候就换成了TxCaptchHelper, 可是在其弹出之后没有找到ticket可以返回的地方。想要来求助。
![198B234A1C566A71D5D0C9820F0CB8EA](https://user-images.githubusercontent.com/58930956/119501847-d2d9c600-bd9b-11eb-8ed0-904057c45940.png)

**Comment by Karlatemp** [comment]
下面那行就是输入 ticket 的地方
